CMakeLists.txt

```diff
@@ -4,17 +4,40 @@ project (SyclSTL)
 enable_testing()
 
 option(PARALLEL_STL_BENCHMARKS "Build the internal benchmarks" OFF)
+option(USE_COMPUTECPP "Use ComputeCPP" ON)
```

Comment:
if you set this option to OFF you will use triSYCL instead, I agree that the variables name might not be the most meaningful.